Relabel Updates to Resource Center (#809)

0xemc opened the pull request:
## Description

Refactor "Updates" navigation item to be "Resource Center" 

## Related Ticket

Resolves #809

## Changes

| Before  | After  |
|---------|--------|
|<img width="413" alt="image" src="https://user-images.githubusercontent.com/7104689/205747206-bdf042e9-927f-4018-afd7-dd7f77af98b8.png">|<img width="276" alt="image" src="https://user-images.githubusercontent.com/7104689/205747165-4b71cbb1-c7e2-4ce5-817f-5af74cc7fe9e.png">|


## Checklist

- [x] Building the site with `npm run build-site` works without errors
- [x] Building the app with `npm run build-app` works without errors
- [x] I formatted JS and TS files with running `npm run format-all`

ladytrekker reviewed (approved):
Thanks

Here in this PR it makes sense to run extract-strings:dev
because this time you really changed translation IDs (deleted old ones, added new => replaced IDs)

In all other cases ff IDs are not changed (not added, replaced or deleted) => no need to extract strings

0xemc commented:
> Thanks
> 
> Here in this PR it makes sense to run extract-strings:dev because this time you really changed translation IDs (deleted old ones, added new => replaced IDs)
> 
> In all other cases if IDs are not changed (not added, replaced or deleted) => no need to extract strings

@ladytrekker Ah ok, are these translations auto generated as part of the vercel build anyway? It would be good to automate this I think to avoid mistakes like mine.
